### ISSUE TITLE
Gap & teams in data 

### ISSUE THREAD

**Issue by loicy** [For-The-Wolf/zwift-scrape#4]
Hi again !

Is it possible to adapt the script to have the gap between the riders?  Possibly also the team ;)

Sorry, I don't know much about python... ;(

**Comment by For-The-Wolf** [comment]
The gap between riders should be easy to calculate from the output but I can add it as a default column too :)  I'll look into adding teams :)

**Comment by loicy** [comment]
The problem with calculating the gap is that the time is put into MS and takes the general time, so the gap is always 1 second, 2 seconds, etc.

**Comment by loicy** [comment]
The script only takes the data when the category is finished? I just tested on a race but it only takes the data when a category is finished.

Is there a possibility to scrap the data even when the race is in progress?

Actually, I use this script to scrap the data for a livestream.

**Comment by For-The-Wolf** [comment]
Yes it only works on a completed race as that was my use case - I should have some time this week to look at your first feature request, however as I am writing my thesis I don't really have the capacity to look into making it work on ongoing races, sorry!

You are of course welcome to fork the repo and have a go at making this work yourself and I'd be happy to pull it in if it works :)
